### ISSUE TITLE
nv-file-select with multiple attribute problem

### ISSUE THREAD

**Issue by kamilkp** [nervgh/angular-file-upload#525]
Hi.

I see in your source code that if an `input` has `multiple` attribute; after the file(s) are selected in the file picker you clear out that input's value and also replace it with a clone of itself - this was introduced [in this commit](https://github.com/nervgh/angular-file-upload/commit/13da2c0c214e5cf238a65006c7f3761ac30cbeb9)

The problem is that cloning an element and passing a `true` as an argument to `clone` copies the event handlers **only and only if** we are on jQuery. It is broken on jqLite. So this results in a behavior that after selecting some file in the filepicker, the input is essentially useless (the clone doesn't have the `onChange` event listener anymore).

I could provide a jsfiddle, but one quick glance at angular's jqLite implementation of `clone` should be enough:

```
function jqLiteClone(element) {
  return element.cloneNode(true);
}
```

**Comment by jmkenz** [comment]
Yes I've run into this same issue and took me a while to narrow it down to needing jQuery - thank you for confirming the reason behind this.  Will this change be going into the repo?

**Comment by lordnox** [comment]
Just found and tracked the same bug down.
jQuery does a lot of things during cloning... there will not be an easy replacement for the .replaceWith method.

I'd propose removing it when on jQLite.

How did you resolve the issue?

**Comment by jmkenz** [comment]
So far my only solution has been to use jQuery I'm afraid. That's the only thing in my app that needs it. I wonder if I could extract just the .clone method from jQuery and include it in my app?

**Comment by hikirsch** [comment]
it seems if i add a this.bind after the clone it works. to avoid the double binding based on jquery, would it be acceptable to clone just the html and then rebind the element instead? i really don't want this to be the only reason i have jquery in my project.
